### DISCUSSION TITLE
Welcome to box86 Discussions!

### DISCUSSION THREAD

**Discussion by ptitSeb** [ptitSeb/box86#264] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chills340** [comment]
I am chills340 and I discovered BOX86 about a year ago when I bought my first Raspberry PI (PI4). I love this community and what it has done to make x86 on ARM possible. I'm not a coder so I like to help out the best I can by testing the occasional program and reporting any issues encountered so fixes can be applied. If there's anything I can provide input on i'll try to assist others as well.
